run-acceptance-tests-local-jenkins.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sbt -Dbrowser="remote-chrome" -Denvironment="local" "testOnly ndrcApplication.suites.RunSuite"
```

Comment:
RED: This is the wrong script.  the -local version shoud use chrome.  The run-acceptance-test-qa-jenkins.sh should be renamed to run-acceptance-test-jenkins.sh and the build job should trigger that script instead of this one

Reply:
Ignore the above - missed this was a new file